docs/api.rst

```diff
@@ -818,6 +818,21 @@ Integrations
     :param payload: The raw event payload data.
     :type payload: :class:`RawIntegrationDeleteEvent`
 
+Audit Logs
+++++++++++
+
+.. function:: on_audit_log_entry_create(entry)
```

Comment:
There should be a warning about the cache reliance here, since it's not receiving a full audit log object but only a single entry.
It therefore doesn't have access to all the [additional objects](https://discord.com/developers/docs/resources/audit-log#audit-log-object-audit-log-structure) usually provided by the API, so `AuditLogEntry.target` more frequently ends up as just an `Object` with the ID instead of a full webhook object or w/e

Reply:
What do you think about [this](527e9b946b70b91c8cd77cc87cb187fb961a2cff)?

Reply:
Would be good to go a bit more into the cache aspect, "Discord limitation" sounds fairly vague. Seems good other than that :)

```rst
    .. warning::
        This scope of data in this gateway event is limited, which means it is much more
        reliant on the cache than :meth:`Guild.audit_logs`.
        Because of this, :attr:`AuditLogEntry.target` will frequently
        be of type :class:`Object` instead of the respective model.
```

Reply:
I knew I could count on you, can you suggest it, so I can commit it directly, please

Reply:
done, thanks :pray: